patientsearch/config.py

```diff
@@ -139,3 +139,5 @@ def load_json_config(potential_json_string):
 PROJECT_NAME = os.getenv("PROJECT_NAME", "COSRI")
 REQUIRED_ROLES = json.loads(os.getenv("REQUIRED_ROLES", "[]"))
 UDS_LAB_TYPES = json.loads(os.getenv("UDS_LAB_TYPES", "[]"))
+ACTIVE_PATIENT_FLAG = os.getenv("ACTIVE_PATIENT_FLAG")
```

Comment:
these will have a value of `None` if not defined, which works, but I see most use assumes a boolean?  might be better to pass `False` as the default value in the `os.getenv()` calls.